### ISSUE TITLE
View.xml is inheriting image sizes from parent (so an optional field is replaced by the value of parent)

### ISSUE THREAD

**Issue by quisse** [magento/magento2#12250]
I want to resize my images to a width of 250px and keep the aspect ratio (so I'm not adding a height tag). Instead of resizing the image to the full height of the image, Magento takes the height for that image from the parent theme's view.xml. So ["If your theme does not contain a view.xml configuration file, it will be inherited from the parent theme. **If you add the <theme_dir>/etc/view.xml file in your theme, _it overrides the parent’s file._**"](http://devdocs.magento.com/guides/v2.0/frontend-dev-guide/themes/theme-inherit.html#override-viewxml-file) is a lie?

### Preconditions

1. Magento 2.1.9
2. PHP 7.0.25

### Steps to reproduce

1. Copy view.xml from a parent theme and add it to new theme
2. Remove `<height>` tag from image
3. Resize images
```
            <image id="product_page_image_small" type="thumbnail">
                <width>250</width>
                <aspect_ratio>true</aspect_ratio>
                <frame>false</frame>
            </image>
```

### Expected result

1. An image with 250px width which kept the aspect ratio without adding 

### Actual result

1. An image with 250px width and 90px height (as defined in the parent view.xml)
2. ![image](https://user-images.githubusercontent.com/2648853/32791191-b45494b8-c960-11e7-899c-ae3fac2c0a6d.png)

**Comment by magento-engcom-team** [comment]
@quisse, thank you for your report.
We've created internal ticket(s) MAGETWO-84055 to track progress on the issue.

**Comment by magento-engcom-team** [comment]
Hi @quisse. Thank you for your report. 
The issue has been fixed in magento/magento2#14537 by @quisse in 2.2-develop branch
Related commit(s):
 - [ca47981c00d1a4d1803520108d5a152d7b3d584a](https://github.com/magento/magento2/commit/ca47981c00d1a4d1803520108d5a152d7b3d584a)
 - [951d5e8bcad09fec5619ad5da582b67025f6e914](https://github.com/magento/magento2/commit/951d5e8bcad09fec5619ad5da582b67025f6e914)
 - [96c28a6238559574fa5ba791b080793d794aacef](https://github.com/magento/magento2/commit/96c28a6238559574fa5ba791b080793d794aacef)

The fix will be available with the upcoming 2.2.7 release.

**Comment by sidolov** [comment]
Hi @quisse. Thank you for your report. 
The issue has been fixed in magento/magento2#17439 by @quisse in 2.1-develop branch
Related commit(s):
 - [715d31e4e0507c34fcf91a2c64d9d306e76f070d](https://github.com/magento/magento2/commit/715d31e4e0507c34fcf91a2c64d9d306e76f070d)
 - [9c0d84f3819dbbd2d5669abb4477b52feb2b52ec](https://github.com/magento/magento2/commit/9c0d84f3819dbbd2d5669abb4477b52feb2b52ec)
 - [fa03c607d4465d75f814da04dc688873d5319de5](https://github.com/magento/magento2/commit/fa03c607d4465d75f814da04dc688873d5319de5)

The fix will be available with the upcoming 2.1.16 release.

**Comment by sidolov** [comment]
Hi @quisse. Thank you for your report. 
The issue has been fixed in magento/magento2#17725 by @nmalevanec in 2.3-develop branch
Related commit(s):
 - [d6c7459b26ce2e4e24ef2bbf8547e2790c01324c](https://github.com/magento/magento2/commit/d6c7459b26ce2e4e24ef2bbf8547e2790c01324c)
 - [f2a1c08a401a338fa93f6f35aea34a9643ae811e](https://github.com/magento/magento2/commit/f2a1c08a401a338fa93f6f35aea34a9643ae811e)
 - [240e18df3e108c5bb244ce01945f8e9415675fb4](https://github.com/magento/magento2/commit/240e18df3e108c5bb244ce01945f8e9415675fb4)

The fix will be available with the upcoming 2.3.0 release.

**Comment by gwharton** [comment]
I realise this is old and long go closed, but anyone know if there is there a way of preventing the inheritance of entire images from the parent view.xml files. It would appear if I remove unrequired image types from my view.xml, they just get added from the parent.
